src/goCheck.ts

```diff
@@ -45,6 +45,37 @@ export function notifyIfGeneratedFile(e: vscode.TextDocumentChangeEvent) {
 			}
 		});
 	}
+
+	let settingsBuildTags = goConfig['buildTags'] ? goConfig['buildTags'].split(' ') : null;
+	let fileBuildTags = [];
+	const openSettings = {title: 'Open settings'};
+
+	for (let i = 0; i < e.document.lineCount; i++) {
+		if (e.document.lineAt(i).isEmptyOrWhitespace) {
+			break;
+		}
+		let foundBuildTags = e.document.lineAt(i).text.match(/^\/\/\s*\+build\s+(.+)$/)[1].split(' ');
```

Comment:
When there is no match, the `match` function returns `null`. Accessing the array member on `null` will throw error